<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quiz Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_question_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_quiz_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Quiz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quiz_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Quiz Title</string>
    </property>
   </widget>
   <widget class="QComboBox" name="difficulty">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Easy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="question_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Question Text</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="option1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Option 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="option2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Option 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="option3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Option 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="option4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Option 4</string>
    </property>
   </widget>
   <widget class="QComboBox" name="correct_option">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Option 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Option 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Option 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Option 4</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="explanation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Explanation</string>
    </property>
   </widget>
   <widget class="QListWidget" name="question_list">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>361</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
